Problem statement and other additions (#34)
MrLongbottom/P9-Rapport: sections/introduction.tex

When evaluating search engines, manual labeling of documents is usually used to measure the performance of models\cite{yang2009topic}\cite{Tang2008}.
Since the topics, that is created by the \gls{lda}, is a distribution between of topics over documents, we can use these to evaluate a search algorithm's retrieval performance. 

Our goal is thus to improve news article search by using a combination of \gls{lda}, \gls{lm}, and PageRank.
For our experiments we use the Nordjyske database of news articles as our dataset.
We also want to investigate whether an automatic evaluation measure can be used to evaluate a search engine.
Here we want to look into whether it makes sense to make an evaluation metric based on whether search results are in a given topic.
Finally, since the news article dataset does not include connections between the articles, how PageRank can be run on this needs to be investigated.
With these goals we look at the following questions:

\begin{quote}

> 	\emph{How can search queries be evaluated based on specific documents or topics?}

Reviewer: ```suggestion
	\emph{How can different infromation retrival techniques be evalueted based on search queries?}
```
tror det er mere i denne retning, men der er så også muligvis nogle underpunkter:
- hvordan generer man queries
- hvordan sørger man for at lave evaluering på en måde som ikke bare laver tf-idf til det bedste

Reviewer: ```suggestion
	\emph{How can different information retrieval techniques be evaluated based on search queries?}
	\begin{itemize}
		\item How can queries be generated for a dataset?
		\item How to evaluate in a way that favors abstraction, rather than just word frequency?
	\end{itemize}
```

> 	\emph{How can search queries be evaluated based on specific documents or topics?}
> \end{quote}

Author: ```suggestion
	\emph{How can different information retrieval techniques be evaluated based on search queries?}
	\begin{itemize}
		\item \emph{How can queries be generated for a dataset?}
		\item \emph{How can evaluation be done in a way that favors abstraction, rather than word frequency?}
	\end{itemize}
\end{quote}
\vspace{0.1 cm}
```

\begin{quote}

> 	\emph{How can PageRank be used on a news article dataset where connections between articles are not obvious?}

Reviewer 2: ```suggestion
	\emph{How can PageRank be used on a news article dataset graph where edges between articles are not obvious?}
```

> 	\emph{How can PageRank be used on a news article dataset where connections between articles are not obvious?}

Reviewer 2: ```suggestion
	\emph{How can PageRank be used on a news article dataset where connections do not exist?}
```

> 	\emph{How can PageRank be used on a news article dataset where connections between articles are not obvious?}

Reviewer: ```suggestion
	\emph{How can PageRank be used on a dataset with no connections between documents?}
```

When evaluating search engines, manual labeling of documents is usually used to measure the performance of models\cite{yang2009topic}\cite{Tang2008}.
Since the topics, that is created by the \gls{lda}, is a distribution between of topics over documents, we can use these to evaluate a search algorithm's retrieval performance. 

Our goal is thus to improve news article search by using a combination of \gls{lda}, \gls{lm}, and PageRank.
For our experiments we use the Nordjyske database of news articles as our dataset.
We also want to investigate whether an automatic evaluation measure can be used to evaluate a search engine.
Here we want to look into whether it makes sense to make an evaluation metric based on whether search results are in a given topic.
Finally, since the news article dataset does not include connections between the articles, how PageRank can be run on this needs to be investigated.
With these goals we look at the following questions:

\begin{quote}

> 	\emph{How can different information retrieval techniques be evaluated based on search queries?}

Reviewer 2: ```suggestion
	\emph{How can information retrieval techniques be evaluated based on search queries?}
```
